lib/Conversion/ImportVerilog/FormatStrings.cpp

```diff
@@ -138,6 +138,15 @@ struct FormatStringParser {
         emitLiteral(lit->getValue());
         return success();
       }
+
+      if (slang::ast::NamedValueExpression::isKind(arg.kind)) {
+        if(options.width) 
+          return mlir::emitError(loc)
+                 << "string format specifier with width not supported";
+        
+        fragments.push_back(context.convertRvalueExpression(arg));
+        return success();
+      }
```

Comment:
This seems unrelated to the stream concat operation. Could you remove this change from this PR (and maybe open a separate PR for this, with a few tests)? Thanks 😃